src/utils/constants.ts

```diff
@@ -0,0 +1,11 @@
+export const DEFAULT_BLOCK_LOGS_LIMIT = 10000 // Should never be 0
```

Comment:
As you can see is already with a default on the environment variables, this can be safely deleted